README.md

```diff
@@ -1,30 +1,57 @@
-# SDP Team 3
+# Blindly
 
 [![Build Status](https://api.cirrus-ci.com/github/BlindlyTeam/Blindly.svg)](https://cirrus-ci.com/github/BlindlyTeam/Blindly)
 [![Maintainability](https://api.codeclimate.com/v1/badges/e05cc8fea21231b94568/maintainability)](https://codeclimate.com/github/BlindlyTeam/Blindly/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/e05cc8fea21231b94568/test_coverage)](https://codeclimate.com/github/BlindlyTeam/Blindly/test_coverage)
 
+Blindly is a dating application designed to meet new people. But what really makes it stands out, is that no pictures are allowed here. 
+At Blindly we believe that love is blind. We value deep meaningful connections. That’s why you’ll use your beautiful voice to present yourself to the world!
+Once you've found your match, you will be able to communicate with them in the integrated chat and plan for a date.  
 
-## Team / App name:
-Blindly
+## Features:
 
-## Description:
-Blindly is a dating app designed to meet new people. No profile pictures are allowed here, the app relies solely on voice. After all, love is blind! Once you’ve matched with someone,  you can go ahead and text them.        
+### Login
+<img src="screenshots/login.png" width="200"> <img src="screenshots/house_rules.png" width="200">

```

Comment:
In the login page I noticed that Sign In ... are not aligned, would it be possible to fix it maybe?

Reply:
Indeed, I believe they are centered in the pill shape button instead of at a fixed distance from the logo to its left
I don't know if it would be better?

Reply:
Oh yeah could be, for me it would make more sense if they are left aligned but it's a bit up to discussion so If you feel they're fine, they stay that way.